CHANGELOG.md

```diff
@@ -4,6 +4,9 @@ Releases
 v1.10.0 (unreleased)
 -------------------
 
+-   Thrift: UberFx-compatible modules are now generated for each service inside
+    a subpackage. Disable this by passing a `-no-fx` flag to the plugin.
```

Comment:
I would argue the usability barriers introduced by making this non-default behavior aren't worth the cleanup you get by making it opt-in. Go will strip out any unused code in the final binaries anyhow and like @abhinav says there aren't any extra dependencies being introduced here.